dialect/sql/schema/schema.go

```diff
@@ -647,9 +647,7 @@ func Dump(ctx context.Context, dialect, version string, tables []*Table, opts ..
 				}
 				return ""
 			}(),
-			func() string {
-				return t.Name
-			}(),
+			t.Name,
```

Comment:
@masseelch Not related to this PR, but the linter is falling.

Reply:
I'll fix it with this PR :+1: